constructor/osx/update_path.sh

```diff
@@ -4,7 +4,8 @@
 
 # $2 is the install location, which is ~ by default, but which the user can
 # change.
-set -x
+set -eux
```

Comment:
`-o pipefail`?

Reply:
sh vs bash... so pipefail not supported.... and there are good reasons to stick with sh on osx as bash does not receive updates there due to GPL3 see https://www.reddit.com/r/bash/comments/393oqv/why_is_the_version_of_bash_included_in_os_x_so_old/